plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -113,12 +111,7 @@ private static IFile createJavaSourceFiles(IProject project, AppEngineProjectCon
     Map<String, String> properties = new HashMap<>();
     properties.put("package", Strings.nullToEmpty(packageName)); //$NON-NLS-1$
 
-    boolean servlet25 = isStandardProject && isStandardJava7RuntimeSelected(config);
-    if (servlet25) {
-      properties.put("servletVersion", "2.5"); //$NON-NLS-1$ //$NON-NLS-2$
-    } else {
-      properties.put("servletVersion", "3.1"); //$NON-NLS-1$ //$NON-NLS-2$
-    }
+    properties.put("servletVersion", "3.1"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Could we rename isStandardJava7RuntimeSelected --> isServlet25Selected and have it return false for now? That will make it easier to add a servlet version pop-up in the future if we like.